### PR TITLE
Implement Alert Banner Component #20277

### LINKED ISSUE #20277: Implement Alert Banner Component
Designs:

https://www.figma.com/design/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?node-id=38825-99486&t=UYzgpAKvqtzSVzl7-0

### PR DESCRIPTION
**Pull request by mmilad75**
fixes #20277 

https://github.com/status-im/status-mobile/assets/55688834/525d84a2-57c2-419e-ab86-b2f3bf910e7d

### CODE REVIEW

File: src/quo/core.cljs [hunk 1]
```diff
@@ -203,6 +204,7 @@
 
 ;;;; Banner
 (def banner quo.components.banners.banner.view/view)
+(def alert-banner quo.components.banners.alert-banner.view/view)
```
Comment (J-Son89) [inline src/quo/core.cljs:207]: please keep alpha sorted! 🙏

File: src/quo/components/banners/alert_banner/view.cljs [hunk 2]
```diff
@@ -0,0 +1,44 @@
+(ns quo.components.banners.alert-banner.view
+  (:require [quo.components.banners.alert-banner.schema :as component-schema]
+            [quo.components.banners.alert-banner.style :as style]
+            [quo.components.buttons.button.view :as button]
+            [quo.components.icon :as icon]
+            [quo.components.markdown.text :as text]
+            [quo.foundations.colors :as colors]
+            [quo.theme :as quo.theme]
```
Comment (J-Son89) [inline src/quo/components/banners/alert_banner/view.cljs]: nit [quo.theme]`

**Review by J-Son89 — APPROVED** [review]

File: src/quo/components/banners/alert_banner/view.cljs (hunk 2 above)
Comment (mmilad75) [reply src/quo/components/banners/alert_banner/view.cljs]: Done 👍

File: src/quo/core.cljs (hunk 1 above)
Comment (mmilad75) [reply src/quo/core.cljs:207]: Done

**Review by vkjr — APPROVED** [review]

**Review by ulisesmac — APPROVED** [review]
Good job @mmilad75 !

Please make sure to ask for a design review :+1:

**Review by ilmotta — APPROVED** [review]

**Comment by qoqobolo** [comment]
Hey @mmilad75, thanks for your work 🙏
Could you clarify why this PR needs manual QA, please? 
As far as I can see, a new component has only been implemented here, in such cases, according to the new process, it is enough to send a PR for a design review. Or was the existing code (used in the app) changed here too?

**Comment by J-Son89** [comment]
@

> Hey @mmilad75, thanks for your work 🙏 Could you clarify why this PR needs manual QA, please? As far as I can see, a new component has only been implemented here, in such cases, according to the new process, it is enough to send a PR for a design review. Or was the existing code (used in the app) changed here too?

No manual QA is needed here @mmilad75/ @qoqobolo . Just a design review 👍

**Comment by mmilad75** [comment]
@qoqobolo @J-Son89 Sorry I must've added by mistake. Thanks for correcting it

**Review by Francesca-G — COMMENTED** [review]
@mmilad75 here's the [review](https://www.figma.com/design/e9woQowfaIOCBrXS1jgkhZ/Review-Components-%2F-Mobile?node-id=249-1735&t=fSTOMvFsncGKRzNX-1) 🙏

**Comment by mmilad75** [comment]
> @mmilad75 here's the [review](https://www.figma.com/design/e9woQowfaIOCBrXS1jgkhZ/Review-Components-%2F-Mobile?node-id=249-1735&t=fSTOMvFsncGKRzNX-1) 🙏

Thanks @Francesca-G The issues are fixed now. Please test again

**Review by Francesca-G — APPROVED** [review]
Looks good now 👍

**Pull request merged**